docs/install/configuration.md

```diff
@@ -43,7 +43,9 @@ If both are defined, then the **environment variables** will take priority.
     <td><h6 class="properties">ZEPPELIN_PORT</h6></td>
     <td><h6 class="properties">zeppelin.server.port</h6></td>
     <td>8080</td>
-    <td>Zeppelin server port</td>
+    <td>Zeppelin server port. </br>
```

Comment:
You're right! I'll remove dot as soon as possible. 
Thanks for properly catching this!